nengo/networks/tests/test_product.py

```diff
@@ -35,3 +35,100 @@ def test_sine_waves(Simulator, plt, seed):
     plt.xlim(right=t[-1])
 
     assert rmse(AB[:len(offset), :], sim.data[p][offset, :]) < 0.2
+
+
+class HilbertCurve(object):
```

Comment:
Yeah, it's being used as an input function, so `nengo.utils.functions` seems appropriate.